### ISSUE TITLE
Duplicated screen output funannotate compare

### ISSUE THREAD

**Issue by ferninfm** [nextgenusfs/funannotate#491]
A trivial report....

funnanote compare v 1.8.1 has a faulty screen output with duplicated information. It is consistently so, I do not find the problem in the code. Other modules do not show this behaviour... (i.e. funannotate annotate).

Otherwise it seems to work. I might just need new glasses...

```
-------------------------------------------------------
[10:16 AM]: OS: linux2, 32 cores, ~ 198 GB RAM. Python: 2.7.15
-------------------------------------------------------
[10:16 AM]: OS: linux2, 32 cores, ~ 198 GB RAM. Python: 2.7.15
[10:16 AM]: OS: linux2, 32 cores, ~ 198 GB RAM. Python: 2.7.15
[10:16 AM]: Running 1.8.1
[10:16 AM]: Running 1.8.1
[10:16 AM]: Now parsing 23 genomes
[10:16 AM]: Now parsing 23 genomes
[10:16 AM]: working on Pyrenodesmia erodens
[10:16 AM]: working on Pyrenodesmia erodens
```

**Comment by nextgenusfs** [comment]
Strange. Have never seen this before. Is this in the terminal output or the logfile?

**Comment by nextgenusfs** [comment]
Oh this is only in compare. I think I know what I did. Will fix in a min.

**Comment by nextgenusfs** [comment]
Okay, let me know if this fixes https://github.com/nextgenusfs/funannotate/commit/2520c60e35fa4f8e8956d80b93e8b43ddcdf0143.

**Comment by ferninfm** [comment]
Sorry I lost track of this...
It is solved as of v1.8.2
